### ISSUE TITLE
Specify the Account

### ISSUE THREAD

**Issue by vinceyap88** [rterp/SumZeroTrading#25]
Hi,
   Is there a way to specify the account number before starting the trading via the API?
:)Vincent Yap

**Comment by rterp** [comment]
hmmm, I don't think that is there yet. What are you intending to use it for?  Do you have a master account with multiple sub accounts?

**Comment by vinceyap88** [comment]
I intend to create a family account. I have added the account handling in InteractiveBroker.java and TraderOrder.java. You can review the diff and merge it if it is
[InteractiveBrokersBroker.java.diff.txt](https://github.com/rterp/SumZeroTrading/files/348804/InteractiveBrokersBroker.java.diff.txt)
[TraderOrder.java.diff.txt](https://github.com/rterp/SumZeroTrading/files/348805/TraderOrder.java.diff.txt)

 correct.

**Comment by rterp** [comment]
Changes look good @vinceyap88   If you want you can create a pull request to submit your changes to be merged into the mainline.  Also, if there are any unit tests touching the InteractiveBrokersBroker and the TradeOrder classes, it would be good to test the new account number functionality.
